src/EditorFeatures/CSharpTest/ExtractMethod/ExtractMethodTests.cs

```diff
@@ -10371,8 +10372,8 @@ public void ExtractMethodCommandDisabledInSubmission()
                 var textView = workspace.Documents.Single().GetTextView();
 
                 var handler = new ExtractMethodCommandHandler(
+                    exportProvider.GetExportedValue<IThreadingContext>(),
```

Comment:
Why not GetService like the rest?

Reply:
➡️ `IThreadingContext` is not a workspace service